### PR TITLE
add basic html notification section in a container class

### PR DESCRIPTION
**Pull request by polmih**
add basic html notification section in a container class adjust the user experience (confirmed with Francis).
current-display

with-container

### CODE REVIEW

**Review by duboisp — CHANGES_REQUESTED** [review]
Can you try to remove the alert. The alert pattern is designed to be used inside the page content area, not outside like before the page header. As an alternate solution, can you tryto add CSS to style it like a yellow background banner, like bg-warning, and keep the semantic markup.

**Review by duboisp — COMMENTED** [review]
Try to improve the selector instead of using a new CSS class,

Like:
```
.wb-disable #wb-tphp .bg-warning h2::before
```

**Review by duboisp — CHANGES_REQUESTED** [review]
Review the CSS selector

**Comment by duboisp** [comment]
as discussed @GormFrank please check if we still avoid this failure: https://www.w3.org/WAI/WCAG22/Techniques/failures/F24

File: src/base/accessibility/_base.scss [hunk 1]
```diff
@@ -68,6 +65,15 @@
...
 	}
+
+	#wb-tphp + section.bg-warning h2::before{
+		color: $label-alert-warning-border-icon-color;
+		content: "\e107";
+		display: inline-block;
+		font-family: "Glyphicons Halflings";
+		margin-left: -1.3em;
+		position: absolute;
+	}
```
Comment (GormFrank) [inline src/base/accessibility/_base.scss:76]: As discussed:
```suggestion
	#wb-tphp + section.bg h2 {
		margin-left: 1.3em;
	
		&::before {
			color: $label-alert-warning-border-icon-color;
			content: "\e107";
			display: inline-block;
			font-family: "Glyphicons Halflings";
			margin-left: -1.3em;
			position: absolute;
		}
	}
```

**Review by GormFrank — CHANGES_REQUESTED** [review]

File: src/base/accessibility/_base.scss (hunk 1 above)
Comment (polmih) [reply src/base/accessibility/_base.scss]: Done, thanks

**Review by GormFrank — APPROVED** [review]

**Review by duboisp — APPROVED** [review]

**Pull request merged**